modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -1340,19 +1445,34 @@ GrainTracker::boundingRegionDistance(std::vector<MeshTools::BoundingBox> & bboxe
   return min_distance;
 }
 
+unsigned int
+GrainTracker::getNextUniqueID()
+{
+  /**
+   * Get the next unique grain ID but make sure to respect
+   * reserve ids.
+   */
+  while (++_max_curr_grain_id <= _reserve_grain_first_idx + _n_reserve_ops)
```

Comment:
I'll add a comment to explain what I'm doing there so it doesn't trip anyone else up. It's easier to assign the +1 to the `_reserve_grain_first_idx` variable so I don't have to do it every time I need to report that information.